main.py

```
@@ -931,34 +928,186 @@ def uninstall_plugin(self):
+                "Go to Search",
+                lambda: self._tutorial_go_to_tab("Search"),
+            ),
+            (
+                "You're all set",
+                f"""<p>That covers the core workflow. A few more things worth knowing:</p>
+<ul>
+  <li><b>Bulk Create</b> tab — create many jobs at once from a CSV file</li>
+  <li><b>Help → Run Setup Wizard</b> — re-run first-time configuration any time</li>
+  <li><b>ITAR mode</b> — keep controlled documents in a separate directory</li>
+</ul>
+<p>If you get stuck, the setup wizard covers directory configuration step by step.</p>""",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)